src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book testBook1 = bookDao.create(new Book("Harry Potter and the Philosophers Stone",
+                BigDecimal.valueOf(9.99)));
+
+        //findById method test
+        Optional<Book> testFindBook = bookDao.findById(1L);
+        if (testFindBook.isPresent()) {
+            Book test1 = testFindBook.get();
+            System.out.println("Find by ID1 book: " + System.lineSeparator() + test1);
+        }
+
+        //Update method test
+        testBook1.setPrice(BigDecimal.valueOf(5.99));
+        bookDao.update(testBook1);
+        System.out.println(System.lineSeparator()
+                + "Suggested price changed from 9.99 to 5.99 on book with ID 1");
+        bookDao.findAll().forEach(System.out::println);
+
+        Book testBook2 = bookDao.create(new Book("Harry Potter and the Chamber of Secrets",
+                BigDecimal.valueOf(10.99)));
+        Book testBook3 = bookDao.create(new Book("Harry Potter and the Prisoner of Azkaban",
+                BigDecimal.valueOf(11.99)));
+        Book testBook4 = bookDao.create(new Book("Harry Potter and the Goblet of Fire",
+                BigDecimal.valueOf(12.99)));
+        Book testBook5 = bookDao.create(new Book("Harry Potter and the Order of the Phoenix",
+                BigDecimal.valueOf(13.99)));
+        Book testBook6 = bookDao.create(new Book("Harry Potter and the Half-Blood Prince",
+                BigDecimal.valueOf(14.99)));
+        Book testBook7 = bookDao.create(new Book("Harry Potter and the Deathly Hallows",
+                BigDecimal.valueOf(15.99)));
```

Comment:
You are creating multiple `Book` objects with the same title. According to the checklist, the `books.title` should not be UNIQUE, which is correct. However, if the current database schema mistakenly has the `books.title` set as UNIQUE, this will cause an SQL exception. Ensure that the schema is set up correctly as per the requirements.